ansible/www-standalone/resources/config/nodejs.org

```diff
@@ -132,11 +197,21 @@ server {
         default_type text/plain;
         open_file_cache_errors on;
 
+        # This rewrite is done to redirect legacy /dist/staging requests to /dist directly
+        rewrite ^/dist/staging/(.*)$ /dist/$1 permanent;
+
         location ~ \.json$ {
             add_header access-control-allow-origin *;
         }
     }
 
+    # This Location directive redirects to the overall available Downloads on this Server
+    # We use `open_file_cache_errors` that allows the NGINX file cache to also cache that
+    # certain files gave 404's, so that it doesn't keep retrying to open this file that doesn't exist
+    # This policy will still follow the expiry time of the file_cache. It is used mostly to avoid our file system
+    # to be hammered by concurrent requests and mitigate situations where the fs stops fulfiling requests
+    # because of maximum concurrent sockets being open.
+    # We use ^~ to tell NGINX to not process any other Location directive or Rewrite after this match
```

Comment:
```suggestion
    # This location directive exposes the download directory, including index listings
    # We use `open_file_cache_errors` here to allow NGINX to also cache metadata for missing files on disk
    # This still follows the expiry time of the open_file_cache. It is used mostly to avoid our file system
    # being hammered by concurrent requests and mitigate situations where the fs stops fulfiling requests
    # because of maximum concurrent sockets being open.
    # We use ^~ to tell NGINX to not process any other location directive or rewrite after this match
```
Very similar wording to above